Docs: SAML docs refactoring (#103450)
grafana/grafana: docs/sources/setup-grafana/configure-security/configure-authentication/saml/configure-saml-with-azuread/_index.md

   - Use this URL in the `idp_metadata_url` field in the `custom.ini` file.

### Generate a client secret

1. In the **Overview** pane, select **Certificates & secrets**.
1. Select **New client secret**.
1. In the **Add a client secret** pane, enter a description for the secret.
1. Set the expiration date for the secret.
1. Select **Add**.
1. Copy the value of the secret. This value is used in the `client_secret` field in the [SAML configuration](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-security/configure-authentication/saml/saml-configuration-options/).

## Configure a Graph API application in Entra ID

While an Entra ID tenant can be configured in Grafana via SAML, some additional information is only accessible via the Graph API. To retrieve this information, create a new application in Entra ID and grant it the necessary permissions.

> [Entra ID SAML limitations](https://learn.microsoft.com/en-us/entra/identity-platform/id-token-claims-reference#groups-overage-claim)

> For the following configuration, the URL `https://localhost` will be used as the Grafana URL. Replace it with your Grafana instance URL.

> ### Create a new Application registration

Reviewer: To match the name with the resource name on the Azure Portal.
```suggestion
### Create a new App registration
```

This app registration will be used as a Service Account to retrieve more information about the user from the Entra ID.

> 1. Go to the [Entra ID portal](https://portal.azure.com/#home) and sign in with your Entra ID account.

Reviewer: ```suggestion
1. Go to the [Azure portal](https://portal.azure.com/#home) and sign in with your Entra ID account.
```

> 1. In the left-hand navigation pane, select the Azure Active Directory service, and then select **App registrations**.

Reviewer: ```suggestion
1. In the left-hand navigation pane, select the Microsoft Entra ID service, and then select **App registrations**.
```

1. Click the **New registration** button.
1. In the **Register an application** pane, enter a name for the application.
1. In the **Supported account types** section, select the account types that can use the application.

> 1. In the **Redirect URI** section, select Web and enter `https://localhost/login/azuread`.

Reviewer: Is this required? @linoman
